buildah-imagestreams/README.md

```diff
@@ -0,0 +1,135 @@
+## Buildah Using Image Streams
+
+The `Task` and `TaskRun` samples here illustrate both how to:
+
+- Have a `Step` in a `Task` use an `ImageStream` reference for its image
+- Have a `Step` push a built image to the OpenShift internal image registry as part of hosting the built image in an `ImageStream`,
+including doing so with the correct certificate so that the push can be performed with TLS verification turned on.
+
+## Preliminary set up
+
+In addition to the actual create of the `Task` and `TaskRun`, the following steps are needed:
+
+- As an administrator to the cluster, create a new `ClusterRole` that allows your `Task` to fetch the internal image registry
```

Comment:
Is this needed?  Given that it's for OpenShift and the `pipeline` service account in every namespace already has sufficient privileges

Reply:
Sorry but the `pipeline` SA does *NOT* have sufficient permissions to read the cert container config map in the `openshift-image-registry` namespace.

Aside from seeing this in testing, if you run `oc adm policy who-can get configmaps -n openshif-image-registry` on your cluster, the `pipeline` SAs are not listed.

Reply:
And hence my note to @nikhil-thomas above about creating a local copy of that CM in the user's namespace.

Where again, openshift builds does this, but only after you create/start a build.

Reply:
@sthaha  is this something to be added to the `pipeline` sa?

Reply:
certs don't work the same way with serviceaccounts as auth tokens ... you don't add the cert to the SA

Again, I think pipelines needs to approximate what we do with v1 builds as I mentioned above